src/s2wasm.h

```diff
@@ -1193,6 +1198,42 @@ class S2WasmBuilder {
         }
       }
     }
+    if (!!startFunction) {
```

Comment:
String performance on large strings are a gain with the `Name` approach, but small strings usually win with the inline approach that most `std::string` implementations use. Names are usually small, so without measurements showing otherwise I'd just stick to `std::string` until we measure a perf issue. There's also the idiomatic argument: C++ APIs work a certain way, `Name` looks different and isn't obvious. This `!!` thing is an example of that.

I went with `Name` since the rest of the codebase does that, but the corners seem sharp from here!